english/see-smore-options/see-smore-options.md

```diff
@@ -0,0 +1,6 @@
+Not sure of what to do, you decide to phone-a-friend for help.  You hit speed dial for your friend, Graham Hershey.  Graham picks up after three rings, but you sense this is not going to end well.  Graham sounds like he's already toasted, and the background noise suggests he's sitting in a bar.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.